### ISSUE TITLE
Stuck in trying pin

### ISSUE THREAD

**Issue by BlackSky079** [drygdryg/OneShot#89]
![Screenshot_20230223-151341_Termux](https://user-images.githubusercontent.com/126153396/220865192-a82bd960-775a-4ddf-8566-567bec07b12a.jpg)

It just stuck in there for hour and hour...I have tried so many Networks and it stays same...I am using a samsung phone with stock rom and kernel...

**Comment by october262** [comment]
is WPS enabled in the router ??

**Comment by BlackSky079** [comment]
Yes...I have also tried with my personal dlink router enabling wps mode and a easy number password...It shows the same...and sorry for my bad English...

**Comment by october262** [comment]
scroll down to the bottom page and try the troubleshooting tips - https://github.com/drygdryg/OneShot

**Comment by BlackSky079** [comment]
I've already tried...still same

**Comment by BlackSky079** [comment]
I want to see how it looks at first when you try to crack a wifi pass and its scanning

**Comment by BlackSky079** [comment]
and is this tool really works nowadays

**Comment by october262** [comment]
it depends, most routers nowadays have been patched to stop this kind of attack.
have you tried oneshot on kali linux to see if it works ??
